<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UniqueInsertCriteriaDialog</class>
 <widget class="QDialog" name="UniqueInsertCriteriaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unique insert criteria</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbSourceTableLabel">
       <property name="text">
        <string>Source:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbSourceTable">
       <property name="text">
        <string>source</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbDestinationTableLabel">
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lbDestinationTable">
       <property name="text">
        <string>destination</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbMatchItems">
     <property name="title">
      <string>Unique check criteria</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="tbRemoveMatchItem">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbAddMatchItem">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="list-add">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbResizeView">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="zoom-fit-best">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="twMatchItems"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UniqueInsertCriteriaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UniqueInsertCriteriaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
